docs/model_system/model_system.md

```diff
@@ -1,4 +1,209 @@
+This allows for single-access lookups rather than recursive traversals.
+
+  ```python
+  # Example: Heterostructure modeling
+  heterostructure = ModelSystem(name='Si/GaAs', is_representative=True)
+  heterostructure.positions = [...]  # All atomic positions
+
+  # Subsystem 1: Silicon layer
+  si_layer = ModelSystem(name='Si_layer', particle_indices=[0, 1, 2, 3])
+  heterostructure.sub_systems.append(si_layer)
+
+  # Subsystem 2: GaAs layer
+  gaas_layer = ModelSystem(name='GaAs_layer', particle_indices=[4, 5, 6, 7, 8, 9])
+  heterostructure.sub_systems.append(gaas_layer)
+```
+
+
+## Structure Representations via `Cell` Sections
+
+### Cell Types and Their Purpose
+
+1. **Original Cell** (`type='original'`)
+   - The cell as provided in the input/calculation
+   - May be any arbitrary choice of unit cell
+   - Contains the lattice vectors used in the actual computation
+
+2. **Primitive Cell** (`type='primitive'`)  
+   - Smallest possible unit cell that tiles the crystal
+   - Contains minimum number of lattice points
+   - Used for certain calculations (e.g., band structures)
+
+3. **Conventional Cell** (`type='conventional'`)
+   - Standardized cell following International Tables conventions
+   - Used for symmetry analysis and crystallographic databases
+   - **Typically used to define symmetry operators:** ["x,y,z", "-x,y,-z", "x+1/2,y+1/2,z", "-x+1/2,y+1/2,-z"] (conventional) vs ["x,y,z", "-x,-y,z", "-x,y,-z", "x,-y,-z"] (primitive).
+  
+<!-- @ndaelman-hu: This is one reading, where we use primitive and conventional to define different symmetry representations.
+The alternative would be to simply label the parsed cell. -->
+
+!!! note Crystal symmetry specification
+    Crystal symmetry operators are dependent on various factors.
+    International table adopt **conventional cell settings** for identify space group operations.
+    These are also used for indexing the space group, which tends to be less situation-dependent.
+     - The same crystal structure gives **different coordinate operations** depending on cell choice
+     - Origin placement and axis orientation affect the algebraic form of operations
+     - Database interoperability requires standardized representations
+
+## Particle Positions: An Open Question
+
+Currently, `ModelSystem` stores particle positions at the top level (`ModelSystem.positions`), but these positions correspond to only **one** of the multiple cell representations. This creates ambiguity about which cell the positions refer to.
+
+```python
+model_system = ModelSystem()
+model_system.positions = [[0.0, 0.0, 0.0], [0.25, 0.25, 0.25]]  # Which cell?
+model_system.cell[0] = Cell(type='original', lattice_vectors=...)    # These positions?
+model_system.cell[1] = Cell(type='primitive', lattice_vectors=...)   # Or these positions?
+model_system.cell[2] = Cell(type='conventional', lattice_vectors=...) # Or these positions?
```

Comment:
From a practical perspective we need the atomic positions for each cell in order to visualize these cells in the GUI. If you anyways run the symmetry analysis to fill the `symmetry` section, then I would say it probably makes sense to store also the atomic positions of the primitive cell/ conventional cell: otherwise we need to run the symmetry analysis again from scratch in the normalizer.